packages/file/index.js

```diff
@@ -26,6 +47,20 @@ let self = {
   name: '@size-limit/file',
   async step60 (config, check) {
     let files = check.bundles || check.path
+
+    if (check.brotli === true) {
+      let isBrotliSupported = gte(process.version, BROTLI_NODE_VERSION)
+
+      if (isBrotliSupported) {
+        check.size = await sum(files, async i => brotliSize(i))
+        return
+      }
+      process.stderr.write(
```

Comment:
Size Limit has a special system for [error messages](https://github.com/ai/size-limit/blob/master/packages/size-limit/size-limit-error.js).

You need to:
1. Add text to `MESSAGES`
2. Use “how to solve” message instead of “what is the problem”. For instance, “Update your Node.js to version >= X to use Brotli”.
3. `throw` an error from a module (it will keep your from requiring `chalk` and make the test much cleaner).

Reply:
Sounds good